engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionPassesNode.java

```diff
@@ -35,9 +39,11 @@
  * TODO: Add diagram of this node
  */
 public class AmbientOcclusionPassesNode extends AbstractNode {
+    public static final ResourceUrn SSAO_URN = new ResourceUrn("engine:ssao");
+    public static final ResourceUrn SSAO_BLURRED_URN = new ResourceUrn("engine:ssaoBlurred");
 
     @In
-    private FrameBuffersManager frameBuffersManager;
+    private DynamicFBOsManager dynamicFBOsManager;
```

Comment:
We named one fbm shadowMapResolutionDependentFBOs. How about we rename this to displayResolutionDependentFBOs and rename the StaticFBM to ImmutableFBOs?